code/datums/components/harvester.dm

```diff
@@ -69,32 +69,32 @@
 		return
 
 	if(!isreagentcontainer(cont) || istype(cont, /obj/item/reagent_containers/pill))
-		to_chat(user, span_rose("[cont] isn't compatible with [source]."))
+		user.balloon_alert(user, "[cont]: not compatible with [source].")
```

Comment:
ballon alerts should use inserts extremely sparingly

The message in balloon alerts should always be as short as possible and not make reference to who or what since the viewer already sees what is being worked with